### ISSUE TITLE
Unable to import books from Amazon

### ISSUE THREAD

**Issue by DrGaellon** [apprenticeharper/DeDRM_tools#2053]
Attempted to import several books to calibre from Kindle. Went to my library, did "download and transfer via USB," saved the files as AZW3. Running Calibre 5.40, DeDRM 7.2.1, Dell Inspiron 15 running Windows 11. Debug file for one of the books is attached (I assume all the others will look similar). Book shows up as still DRMed in calibre.
[tmp287wuxkw.txt](https://github.com/apprenticeharper/DeDRM_tools/files/8465478/tmp287wuxkw.txt)

**Comment by ElleKayEm** [comment]
With that method, you need to choose an E-ink Kindle (not a Fire) when downloading the book from the website.  Then you need to have that Kindle's serial number entered into DeDRM's customization.

**Comment by DrGaellon** [comment]
Ah. I don't have a Kindle device; I read everything through the iOS or PC apps. Does that mean I can't use this method? Is there an alternate way to de-DRM my books?

**Comment by ElleKayEm** [comment]
Yes, the "download & transfer" method only works with DeDRM if you have an E-ink Kindle on your account.  Otherwise, you'll need to download it with Kindle for PC (or Mac) on the same machine and user as calibre with DeDRM.  There are some other things you'll need to do in order to get a file you can decrypt.  Details are here:  https://www.mobileread.com/forums/showthread.php?t=283371
